### PR TITLE
mentioned the fact that LE can take responsibility for the solution architecture

### PR DESCRIPTION
**Pull request by halkar**

### CODE REVIEW

File: Engineering.md [hunk 1]
```diff
@@ -51,9 +51,10 @@ Being technical does not eliminate the need to utilise soft skills, as you will
 #### I am a servant leader for my team
 - I split large difficult problems into smaller more tractable problems that can be tackled by junior team members.
 - I am prioritising the needs of my team members above my own, knowing that their success is our success.
+- I take responsibility for the solution architecture to help my team focus on delivering features and to reduce chore.
```
Comment (mleech) [inline Engineering.md]: Should solution architecture be a individual's responsibility or should it be the team's responsibility? How about something along the lines "I provide strong guidance on solution architecture and help to remove technical blockers"?

Comment (andrewabest) [reply Engineering.md]: I agree with the taking ownership part, but don’t like the implication that it is to allow other people to code without thinking, or to not contribute themselves. Solution architecture isn’t a “chore” as such, and other team members aren’t just there to deliver features.

Comment (halkar) [reply Engineering.md]: Yes, solution architecture isn’t a “chore” but good solution architecture can help reduce it. I don't see much difference between the current text and @mleech version. Happy to use his version.

Comment (halkar) [reply Engineering.md]: Updated

**Pull request merged**